Source/Scene/QuadtreePrimitive.js

```diff
@@ -206,6 +206,20 @@ QuadtreePrimitive.prototype.invalidateAllTiles = function () {
   this._tilesInvalidated = true;
 };
 
+function containsOwnerHeightCallbackData(heightCallbacks, owner) {
+  var found = false;
```

Comment:
Could this be changed to delete the `found` variable, replace `found = true; break;` with `return true;` inside the for-loop, and `return false;` at the end instead of `return found`?